From 66ac8550e2fe2617824befc478d68a70990db382 Mon Sep 17 00:00:00 2001
From: Nicolas Boulenguez <nicolas.boulenguez@free.fr>
Date: Sat, 20 Feb 2021 12:05:59 +0100
Subject: [PATCH 21/21] Document that TARGET builds need a refreshed
 debian/control

It is difficult to let debian/rules refresh debian/control because it
would have to detect if TARGET has changed since last build or clean.
Explanations in README.cross seem sufficient for this use case.
---
 debian/README.cross | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/debian/README.cross b/debian/README.cross
index f118d3b..ea184f8 100644
--- a/debian/README.cross
+++ b/debian/README.cross
@@ -11,6 +11,17 @@ To build a cross-binutils package which is not yet built:
 
     apt-get build-dep binutils
 
+ o Add the binutils-<your-target> package to the control file:
+
+    rm -f stamps/control
+    debian/rules stamps/control TARGET=<your-target>
+
+    This change is permanent and will survive 'make clean'.  If you
+    need to restore the original debian/control file:
+
+    rm -f stamps/control
+    debian/rules stamps/control TARGET=
+
  o Then build the cross-binutils package:
    
     TARGET=<your-target> dpkg-buildpackage -b -uc -us 
-- 
2.30.1

